sdk/azidentity/bearer_token_policy_test.go

```diff
@@ -22,9 +22,11 @@ const (
 )
 
 func defaultTestPipeline(srv azcore.Transport, cred azcore.Credential, scope string) azcore.Pipeline {
-	retryOpts := azcore.DefaultRetryOptions()
-	retryOpts.MaxRetryDelay = 500 * time.Millisecond
-	retryOpts.RetryDelay = 50 * time.Millisecond
+	retryOpts := azcore.RetryOptions{
+		MaxRetryDelay: 500 * time.Millisecond,
+		RetryDelay:    50 * time.Millisecond,
+		TryTimeout:    1 * time.Minute,
```

Comment:
This is no longer the default, is it required for tests to pass?

Reply:
Fixed this, left the values for MaxRetryDelay and RetryDelay, removed TryTimeout. Done.